gnnflow/distributed/dist_graph.py

```diff
@@ -23,6 +28,38 @@ def __init__(self, dgraph: DynamicGraph):
         self._num_edges = 0
         self._max_vertex_id = 0
 
+        self._handle_manager = HandleManager()
+        self._handles = set()
+        self._add_edges_thread = threading.Thread(target=self._add_edges_loop)
+        self._add_edges_queue = Queue()
```

Comment:
Does the add_edges_loop will keep looping throughout the whole training process?

Reply:
Yes. It does not incur much overhead because it will sleep when there is no input data.